docs/config/generate-api-certificates.rst

```diff
@@ -4,7 +4,7 @@
 Generating an API Client Certificate
 ====================================
 
-To send instructions to the control service, whether it is via the API directly, or the CLI, or by any other method, you will need to follow the instructions below to generate an API client certificate:
+To send instructions to the :ref:`Flocker control service <control-service>`, whether it is via the API directly, or the CLI, or by any other method, you will need to follow the instructions below to generate an API client certificate:
```

Comment:
Why is this reference using an explicit text for the link, while `flocker-basics.rst` is implicitly using the title text?  Shoult these be the same, to keep capitalization the same?